### DISCUSSION TITLE
Works great , only if u could have added a way to return the thumbnail too

### DISCUSSION THREAD

**Discussion by yourmomlover** [EchterAlsFake/hqporner_api#1] [Ideas]
yes

**Comment by EchterAlsFake** [comment]
Hi,

I didn't add it in the first place, because you can only get the thumbnail from the page, before you click on the video. For example, if I search for videos, then I can get the thumbnail, because I didn't click it.

But I thought about it and I think I have a solution. I could probably just search for the specified video, but with the exact same title, then check the url, and if it's the same I'd get the thumbnail. It wouldn't be efficient, but the only possible way. I'll work on it, give me some hours :)

**Reply by yourmomlover** [reply]
alr thanks man , u a coding god

**Reply by EchterAlsFake** [reply]
You can now get the thumbnails using video.get_thumbnails()

This will return a list containing 11 items. The first item is the base thumbnail and the other 10 are the thumbnails you see when you hover over a video.

The way it works is by searching for the video. I haven't really tested it with multiple videos, so I don't know how robust this is. Maybe it works for all videos, but I am not sure. You can either directly use Release 1.6 from pypi or install it from GitHub :)

**Reply by yourmomlover** [reply]
i forgot to reply , it works great , thanks for this
